### PR TITLE
Add docker container setup

### PR DESCRIPTION
**Pull request by sgnn7**
https://phabricator.endlessm.com/T16706

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+RUN add-apt-repository ppa:alexlarsson/flatpak
+
+RUN apt-get update && \
+    apt-get dist-upgrade -y && \
+    apt-get install -y apt-transport-https \
+                       curl \
+                       flatpak
+
+
+# Keep in line with requirements.txt and setup.py
```
Comment (lmyllari) [inline Dockerfile:18]: Would it be better to use pip to install the requirements? There would be no need to sync this manually with requirements.txt

Comment (sgnn7) [reply Dockerfile:18]: It's somewhat of a messy toss-up but generally you want the OS-provided libraries in Dockerfile as you can cache that layer easier and are usually more stable releases of code vs PIP. I understand the convenience that PIP provides but in this particular aspect, it's not ideal.

Comment (sgnn7) [reply Dockerfile:18]: Edit: If you'd do prefer the PIP option, I can splice that in

Comment (lmyllari) [reply Dockerfile:18]: No, I think you're right. The requirements shouldn't change often anyway.

**Review by lmyllari — APPROVED** [review]

**Comment by lmyllari** [comment]
Looks good to me. What's your process for reviewing and merging? Wait for @wgreenberg and then merge?

**Comment by sgnn7** [comment]
@lmyllari Just merge it :)

**Pull request merged**